src/api/github.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+export default axios.create({
+    baseURL: 'https://api.github.com/repos/vuejs/vue/issues',
+   
+  })
```

Comment:
```suggestion
export default axios.create({
    baseURL: 'https://api.github.com/repos/vuejs/vue/issues',
})

```

- [ ] remove spacing
- [ ] remove indention
- [ ] add new line at end of file